.github/workflows/build-and-test.yml

```diff
@@ -4,7 +4,7 @@ on:
   push:
     branches: ['main']
   pull_request:
-    branches: ['main', 'next', 'canary']
+    branches: ['main', 'next', 'canary', 'feat_apply_bundler_runtime']
```

Comment:
It feels like this should run on any branch

Reply:
agreed